components/match2/wikis/goals/match_group_input_custom.lua

```diff
@@ -0,0 +1,129 @@
+---
+-- @Liquipedia
+-- wiki=goals
+-- page=Module:MatchGroup/Input/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Operator = require('Module:Operator')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local MatchGroupInputUtil = Lua.import('Module:MatchGroup/Input/Util')
+local Streams = Lua.import('Module:Links/Stream')
+
+local CustomMatchGroupInput = {}
+
+-- called from Module:MatchGroup
+---@param match table
+---@param options table?
+---@return table
+function CustomMatchGroupInput.processMatch(match, options)
+	local winnerInput = match.winner --[[@as string?]]
+	local finishedInput = match.finished --[[@as string?]]
+
+	Table.mergeInto(match, MatchGroupInputUtil.readDate(match.date))
+
+	local opponents = Array.mapIndexes(function(opponentIndex)
+		return MatchGroupInputUtil.readOpponent(match, opponentIndex, {})
+	end)
+
+	local games = CustomMatchGroupInput.extractMaps(match, opponents)
+	local scoreType = 'mapScores'
+	local autoScoreFunction = MatchGroupInputUtil.canUseAutoScore(match, games)
+		and CustomMatchGroupInput.calculateMatchScore(games, scoreType)
```

Comment:
```suggestion
		and CustomMatchGroupInput.calculateMatchScore(games)
```

Reply:
with these two revision is it possible to have the score display taking in-map score on a Bo1 over 1/0? (Similar to the shooting games)
![image](https://github.com/user-attachments/assets/591bb81e-b8a4-420f-9fee-4d2b09bd95c3)

Reply:
Set showBo1 score in Info

Reply:
I've added this update to info but it doesnt trigger the changes: https://liquipedia.net/goals/index.php?title=Module%3AInfo&type=revision&diff=241&oldid=217 (still 1/0)